app/src/ui/history/commit-summary.tsx

```diff
@@ -195,43 +195,56 @@ export class CommitSummary extends React.Component<
     )
   }
 
-  private renderExternalLink() {
-    if (this.props.isLocal) {
-      return null
-    }
-
+  private onShowCommitOptions = () => {
+    let label: string = ''
     const gitHubRepository = this.props.repository.gitHubRepository
-    if (!gitHubRepository) {
-      return null
-    }
 
-    const url = `${gitHubRepository.htmlURL}/commit/${this.props.sha}`
-    const isDotCom = gitHubRepository.endpoint === getDotComAPIEndpoint()
+    if (gitHubRepository) {
+      const isDotCom = gitHubRepository.endpoint === getDotComAPIEndpoint()
+      label = isDotCom ? 'View on GitHub' : 'View on GitHub Enterprise'
+    }
 
-    const label = isDotCom ? 'View on GitHub' : 'View on GitHub Enterprise'
-    const title = isDotCom
-      ? 'View this commit on GitHub'
-      : 'View this commit on GitHub Enterprise'
+    const items: IMenuItem[] = [
+      {
+        label: __DARWIN__ ? 'Revert This Commit' : 'Revert this commit',
+        action: this.onRevertCommit,
+      },
+      { type: 'separator' },
+      {
+        label: 'Copy SHA',
+        action: this.onCopySHA,
+      },
+      {
+        label: label,
+        action: this.onViewOnGitHub,
+        enabled: !this.props.isLocal && !gitHubRepository,
```

Comment:
I believe I had 2 `!!` but for some reason, there is only 1. I can't use `gitHubRepository` in the conditional since it's not technically a bool.